test/task/ensure_test_shop_test.rb

```diff
@@ -13,7 +13,7 @@ def setup
       def test_outputs_if_shop_cant_be_queried
```

Comment:
`test_outputs_if_store_cant_be_queried` ?

Reply:
(I see 'shop' is also in the the test name, and probably other places too, so maybe this can be handled in a separate PR).

Reply:
@paulomarg and I are going to pair on this as a separate task later this week!